wgpu-hal/src/dynamic/mod.rs

```diff
@@ -0,0 +1,67 @@
+use std::any::Any;
+
+use wgt::WasmNotSendSync;
+
+/// Base trait for all resources, allows downcasting via `std::any::Any`.
```

Comment:
suggestion: Let's link!

```suggestion
/// Base trait for all resources, allows downcasting via [`std::any::Any`].
```

Reply:
Implemented with latest force-push.